profiles/templates/auction_worker_defaults.json

```diff
@@ -20,7 +20,7 @@
   "sentinel_cluster_name": "{{sentinel_cluster_name}}",
   "planning_procerude": "{{planning_procerude}}",
   "sandbox_mode": "{{sandbox_mode}}",
-  "with_document_service": {{ with_document_service |default('false') }},
+  "with_document_service": {{ with_document_service|default('false', true) }},
```

Comment:
default('false', true) - що це робить.

Reply:
[Документація](http://jinja.pocoo.org/docs/2.9/templates/#filters) про фільтр **default** каже:

>  `default(value, default_value=u”, boolean=False)`
>  If the value is undefined it will return the passed default value, otherwise the value of the variable:
>    `{{ my_variable|default('my_variable is not defined') }}`
>   This will output the value of my_variable if the variable was defined, otherwise 'my_variable is not defined'. If you want to use default with variables that evaluate to false you have to set the second parameter to true:
>  `{{ ''|default('the string was empty', true) }}`

В нашому випадку в темплейті для **auction_worker_defaults.json** фігурує

`"with_document_service": {{ with_document_service|default('false') }}`,

і ми після підставляння відповідного значення із **profiles/project.cfg**

`with_document_service =` 

отримаємо json із key/value у вигляді 

`"with_document_service":  ""`

що приводить до помилки. Якщо ж в темплейті зробити так:

`"with_document_service": {{ with_document_service|default('false', true) }}`

то значення `with_document_service =` "розв'яжеться" у `false` і в результаті отримаємо json із key/value у вигляді

`"with_document_service": false`

як нам і треба.

Reply:
Виглядає правильно.